ui/src/components/Header/Header.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import './Header.css';
-import sampleProfilePic from '../../img/profile.png';
+import sampleProfilePic from '../../img/hamburger-menu.png';
+import sampleProfilePic2 from '../../img/acm-logo-final.png';
```

Comment:
I think you should give more descriptive names to the icons you import here. Maybe something like:

```suggestion
import hamburger from '../../img/hamburger-menu.png';
import acmVoteLogo from '../../img/acm-logo-final.png';
```